zephyr/CMakeLists.txt

```diff
@@ -393,7 +393,7 @@ zephyr_library_sources(
 	${SOF_SRC_PATH}/schedule/schedule.c
 	${SOF_SRC_PATH}/schedule/dma_single_chan_domain.c
 	${SOF_SRC_PATH}/schedule/dma_multi_chan_domain.c
-	${SOF_SRC_PATH}/schedule/ll_schedule.c
+	${SOF_SRC_PATH}/schedule/zephyr_ll.c
```

Comment:
Lets make this a Kconfig option - both will export the same scheduler type.

Reply:
@lgirdwood you want both to be available with Zephyr - selectable via Kconfig?

Reply:
ah, got you - this in in the zephyr makefile. Please ignore.